gc/base/standard/ConcurrentGCSATB.cpp

```diff
@@ -451,6 +451,8 @@ MM_ConcurrentGCSATB::completeConcurrentTracing(MM_EnvironmentBase *env, uintptr_
 {
 	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
 
+	GC_OMRVMInterface::flushCachesForGC(env);
```

Comment:
Hm, I wonder why is this required. We are within final  STW, and we likely did this as part of acquiring exclusive (I can see concurrentFinalCollection passing true for flush). If we did not somehow, it would be better to enforce it by other means  instead of worrying about it here.

Reply:
I'm not sure if we can assume that caches are flushed in paths other than `concurrentFinalCollection` leading to final STW .

Reply:
Will need to confirm that or look into  `better to enforce it by other means instead of worrying about it here.`